mutiny-core/src/lsp/mod.rs

```diff
@@ -0,0 +1,173 @@
+use crate::error::MutinyError;
+use crate::keymanager::PhantomKeysManager;
+use crate::ldkstorage::PhantomChannelManager;
+use crate::logging::MutinyLogger;
+use crate::node::LiquidityManager;
+use crate::storage::MutinyStorage;
+use async_trait::async_trait;
+use bitcoin::secp256k1::PublicKey;
+use bitcoin::Network;
+use lightning::ln::PaymentHash;
+use lsps::{LspsClient, LspsConfig};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::sync::Arc;
+use voltage::LspClient;
+
+pub mod lsps;
+pub mod voltage;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub enum LspConfig {
+    VoltageFlow(String),
+    LspsFlow(LspsConfig),
```

Comment:
I named it `VoltageFlow` because Flow is the name of the product, I think the LSPS would can just be named `LSPS`

Reply:
oh whoops, I'll change it